srcpkgs/corectrl/template

```diff
@@ -1,16 +1,18 @@
 # Template file for 'corectrl'
 pkgname=corectrl
-version=1.3.1
+version=1.4.1
 revision=1
 build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```

Comment:
Instead of nuking tests this way, use `make_check=no` with a comment explaining why the tests are disabled, and setup test building normally, i.e. conditionally `configure_args+=" -DBUILD_TESTING=ON"` when `XBPS_CHECK_PKGS` is set

Reply:
done

Reply:
Ah nevermind, XBPS_CHECK_PKGS is set independently of make_check, guess we can drop all testing-related code from the template for now

Reply:
idk why this [build](https://github.com/void-land/void-repository/actions/runs/10780044096/job/29894892337) worked fine, but i will remove that condition

Reply:
`# Template file for 'corectrl'
pkgname=corectrl
version=1.4.1
revision=1
build_style=cmake
configure_args="-DBUILD_TESTING=OFF" # requires https://github.com/rollbear/trompeloeil which isn't packaged
hostmakedepends="pkg-config extra-cmake-modules qt5-host-tools qt5-qmake
 kcoreaddons kauth"
makedepends="botan-devel kauth-devel karchive-devel qt5-charts-devel quazip-devel
 qt5-tools-devel libdrm-devel qt5-svg-devel spdlog pugixml-devel"
depends="dbus hicolor-icon-theme qt5-quickcontrols2 qt5-svg glxinfo"
checkdepends="catch2"
short_desc="User-friendly hardware control application"
maintainer="Orphaned <orphan@voidlinux.org>"
license="GPL-3.0-or-later"
homepage="https://gitlab.com/corectrl/corectrl"
changelog="https://gitlab.com/corectrl/corectrl/-/raw/master/CHANGELOG.md"
distfiles="https://gitlab.com/corectrl/corectrl/-/archive/v${version}/corectrl-v${version}.tar.gz"
checksum="dec3471142884e1940daa85bec7dbd4df1bc5a33cf1c523048031d4916fa9acd"
`

next commit will be like this is this ok to merge ?